core/beacon/types.go

```diff
@@ -157,34 +138,43 @@ func ExecutableDataToBlock(params ExecutableDataV1) (*types.Block, error) {
 	if params.BaseFeePerGas != nil && (params.BaseFeePerGas.Sign() == -1 || params.BaseFeePerGas.BitLen() > 256) {
 		return nil, fmt.Errorf("invalid baseFeePerGas: %v", params.BaseFeePerGas)
 	}
+	// Only set withdrawalRoot if it is non-nil. This allows CLs to use
+	// ExecutableDataV2 before withdrawals are enabled by marshaling
```

Comment:
looks like you have a bunch of lingering references in comments like this one to ExecutableDataV2 even though you've dropped that type name.

Reply:
fixed, thanks